<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordListDialog</class>
 <widget class="QDialog" name="WordListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>483</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BruteForce WordList</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabChoose">
      <attribute name="title">
       <string>Choose</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string>Available Wordlist</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="radioButtonDefaultWordlist">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Default WordList</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Choose Wordlist Type:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxDefaultWordlist">
              <property name="minimumSize">
               <size>
                <width>180</width>
                <height>20</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonSpecialWordlist">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Custom WordList</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Choose Wordlist Type:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxSpecialWordlist">
              <property name="minimumSize">
               <size>
                <width>180</width>
                <height>20</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonPredefined">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Predifined</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Choose Wordlist Type:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxPredifinedWordlist"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxCustomWordlist">
         <property name="title">
          <string>New Custom Wordlist</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="createSpecialWordlist">
            <item>
             <widget class="QLineEdit" name="lineEditName">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonCreate">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>23</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>23</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Create</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="InputWidget" name="customWordlist" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonLoadChoosen">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabGenerate">
      <attribute name="title">
       <string>Generate</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QToolBox" name="toolBox">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="pageRanges">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>401</width>
            <height>336</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Ranges</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="QGroupBox" name="groupBoxAZ">
             <property name="title">
              <string>A-Z</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QCheckBox" name="checkBoxAZPermute">
                <property name="text">
                 <string>Permute</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>No# of Permutations</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditAZPermutes"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxNumbers">
             <property name="title">
              <string>Numbers</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>From</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditNumbersFrom"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>To</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditNumbersTo"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Skip</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditNumbersSkip"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxDate">
             <property name="title">
              <string>Date</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>From</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditDateFromDay"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditDateFromMonth"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditDateFromYear"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>To</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditDateToDay"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditDateToMonth"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditDateToYear"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Format</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBoxDateFormat">
                  <property name="editable">
                   <bool>false</bool>
                  </property>
                  <item>
                   <property name="text">
                    <string>dd.mm.yyyy</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>dd-mm-yyyy</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>dd_mm_yyyy</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pagePermute">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>401</width>
            <height>336</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Permutation &amp; Substitution</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <property name="spacing">
            <number>9</number>
           </property>
           <item>
            <widget class="InputWidget" name="generateWordlist" native="true"/>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxPermutations">
             <property name="title">
              <string>Permutations</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>No# of Permutations</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditPermutes"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxSubstitutions">
             <property name="title">
              <string>Substitutions</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QRadioButton" name="radioButtonSubstituteAll">
                <property name="text">
                 <string>Substitute all</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButtonSubstituteOne">
                <property name="text">
                 <string>Substitute one at a Time</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_14">
                <item>
                 <widget class="QComboBox" name="comboBoxSubstituteFrom"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>--&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBoxSubstituteTo"/>
                </item>
                <item>
                 <widget class="QToolButton" name="buttonSubstituteRemove">
                  <property name="minimumSize">
                   <size>
                    <width>70</width>
                    <height>22</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Remove</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label_12">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonGenerate">
           <property name="text">
            <string>Generate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InputWidget</class>
   <extends>QWidget</extends>
   <header>src/widgets/InputWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
